parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -123,6 +122,8 @@ impl Command {
 				let asset_id = Self::convert_token_address(network, gateway, token);
 
 				let mut instructions = create_instructions(origin_location);
+
+				let create_call_index: [u8; 2] = [53, 0];
```

Comment:
As Oak mentioned, we can make it configurable load from some storage.
But I would suspect even this is not necessary as there is also some strict check flow for polkadot/cumulus runtime upgrade to not override an existing call so the call index is unlike to change. So just hardcode here should be fine.

Reply:
That's a good point Ron, which I never realised. The call index should ideally remain the same for all AssetHub runtimes.

If you can confirm that, I'm happy for you to proceed with this PR.

Reply:
> The call index should ideally remain the same for all AssetHub runtimes.

Yes, it's the same for all asset-hub runtimes(e.g. asset-hub-polkado
https://github.com/paritytech/polkadot-sdk/blob/f204e3264f945c33b4cea18a49f7232c180b07c5/cumulus/parachains/runtimes/assets/asset-hub-polkadot/src/lib.rs#L772)